ethereum/gateway-eth-ts/package.json

```diff
@@ -11,6 +11,7 @@
     "postbuild": "rm -f oclif.manifest.json",
     "pretest": "cd ../smart-contract && (yarn local &) && sleep 10",
     "test": "ts-mocha src/*.test.ts src/**/*.test.ts",
+    "posttest": "kill $(lsof -t -i:8545)",
```

Comment:
That seems a little hacky and I'm suprised that hardhat does not support something better here :)